utils/fake_data.py

```diff
@@ -95,6 +95,12 @@ def safe_postal_code(self):
             if state != "AK" and state != "HI":
                 return self.postalcode_in_state(state)
 
+    def safe_uuid(self):
+        """
+        Returns an empty uuid as a string.
+        """
+        return "00000000-0000-0000-0000-000000000000"
```

Comment:
Is there a better way to do this?

Reply:
Yep! There's a constant for this value, so add `ZERO_UUID` to the imports from constants on line 11, and then you can do `return ZERO_UUID` here